src/internal/visual-mode/__tests__/use-visual-mode.test.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useRef } from 'react';
+import { useCurrentMode, useDensityMode } from '../index';
+import { render, screen } from '@testing-library/react';
+import { mutate } from './utils';
+
+jest.mock('../../../environment', () => ({ ALWAYS_VISUAL_REFRESH: false }), { virtual: true });
```

Comment:
This file does not exist in this package. Is it still needed here?

Reply:
Good catch!